packages/client/hmi-client/src/page/Home.vue

```diff
@@ -499,8 +509,9 @@ a {
 	flex-direction: column;
 	justify-content: center;
 	gap: 1rem;
-	border-radius: 4px;
+	border-radius: 8px;
```

Comment:
Please use the `var(--big-border-radius)` that you will create in `_variables.scss`;